src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,50 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  static MIN_HEALTH = 1;
+
+  health = 100;
+
+  static removeFromAlive() {
```

Comment:
The method name `removeFromAlive` suggests it removes a specific animal, but it's actually used to filter out all dead animals. Consider renaming this to better reflect its functionality, like `removeDeadAnimals`.